composer.json

```diff
@@ -1,43 +1,40 @@
 {
-	"name": "silverstripe/dynamodb",
```

Comment:
All lines were touched because I replaced tabs with spaces. May be easier to look through each commit to see what changed here and why.